src/Aspire.Hosting/Dashboard/proto/dashboard_service.proto

```diff
@@ -5,7 +5,7 @@ syntax = "proto3";
 
 package aspire.v1;
 
-option csharp_namespace = "Aspire.ResourceService.Proto.V1";
```

Comment:
I'm torn. I like the idea of the resource server being what the dashboard talks to but it's not a dashboard service

Reply:
It's already called the DashboardService in the proto file. Changing the file name and namespace are lining code up with the actual name.

Reply:
People are using this proto file so it'll break them